doc/nrf/known_issues.rst

```diff
@@ -1475,20 +1542,37 @@ Subsystems
 Bluetooth® LE
 =============
 
-.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
 
-NCSDK-19727: Cannot build the :ref:`peripheral_hids_mouse` sample with security disabled.
```

Comment:
Could you please leave this issue (not remove it)? It has been added together with a bugifx PR; https://github.com/nrfconnect/sdk-nrf/pull/10251 just yesterday. This is a valid issue that is being fixed for NCS 2.3.0.

Reply:
Rebase gone wrong.

Reply:
I'm not removing it. I'm just moving it down a couple of lines (see line 1569).